integration-tests/src/test/java/org/apache/druid/tests/indexer/AbstractITRealtimeIndexTaskTest.java

```diff
@@ -88,25 +89,30 @@ void doTest()
       // the task will run for 3 minutes and then shutdown itself
```

Comment:
update comment to 10 minutes as well?

Reply:
Oops, thanks. Changed back to 5 mins since 10 mins was for testing. I think 3 mins can be too short. Updated the comment as well.